textproc/epubcheck/Portfile

```diff
@@ -4,35 +4,38 @@ PortSystem              1.0
 PortGroup               java 1.0
 
 name                    epubcheck
-version                 1.2
+version                 4.2.2
 categories              textproc
 platforms               darwin
-license                 BSD 
+license                 BSD 3-Clause \"New\" or \"Revised\" License
```

Comment:
I don't think this change to the `license` would be accepted. The guide (https://guide.macports.org/#reference.keywords) essentially specifies using only the abbreviated name of the license(s); I'm not sure if there is a list of what values are allowed, so I don't know if there is something more specific to use than `BSD` in this case.

Reply:
I've picked up all those changes. The license is now "BSD-3-Clause" which is the SPDX designation for the current EPUBCheck license. The GitHub PortGroup is used. The fallback version of java is openjdk11. The kaffe dependency is gone.

I hope the new Portfile is now acceptable.